models/iaf_psc_alpha.h

```diff
@@ -96,10 +96,11 @@ Name: iaf_psc_alpha - Leaky integrate-and-fire neuron model.
   I_e        double - Constant external input current in pA.
   V_min      double - Absolute lower value for the membrane potential.
 
-Remarks:
-  tau_m != tau_syn_{ex,in} is required by the current implementation to avoid a
-  degenerate case of the ODE describing the model [1]. For very similar values,
-  numerics will be unstable.
+Note:
+For very similar values of tau_m and tau_syn_{ex,in} the
+evaluation of the propagator entries P_31 and P_32 is numerically
+unstable. For details, please see IAF_Neruons_Singularity.ipynb in the
+NEST source code (docs/model_details)
```

Comment:
I would write this comment more positive and informative. As it is now, the user may become uncertain about wht precisely happens if the taus are very similar. Here a suggestion:

If tau_m is very close to tau_syn_ex or tau_syn_in, the model will numerically treat the model as if tau_m is equal to tau_syn_ex or tau_syn_in, respectively, to avoid numerical instabilities. The status dictionary provides information about which propagators are used through the /propagator_info entry. For details, please see IAF_Neruons_Singularity.ipynb in the NEST source code (docs/model_details).